firefox_puppeteer/tests/test_windows.py

```diff
@@ -77,11 +77,14 @@ def test_base_window_basics(self):
                           win1.send_shortcut, 'l', acel=True)
 
     def test_base_window_open_close(self):
+        def setUp(self):
+            FirefoxTestCase.setUp(self)
+
         # force BaseWindow instance
         win1 = BaseWindow(lambda: self.marionette, self.browser.handle)
 
         # Open a new window (will be focused), and check states
-        win2 = win1.open_window()
+        win2 = self.browser.open_browser(trigger='menu')
```

Comment:
Your comment for the commit mentions that you have removed the menu trigger, but it is still present. :) So please revert this line to what it was before. We don't want to operate on a browser window in a base window test.